contracts/Makefile

```diff
@@ -4,18 +4,21 @@
 
 test:
 	forge fmt
-	forge test --fork-url https://sepolia.base.org --fork-block-number 16718855
+	forge test --fork-url https://sepolia.base.org --fork-block-number 17180041
```

Comment:
Does changing this block number break all other tests?

Reply:
Forgot to change it, sorry. Fixed.